tox.ini

```diff
@@ -93,14 +92,6 @@ commands =
     cov-!double: pytest --pyargs astropy {toxinidir}/docs {env:MPLFLAGS} --cov astropy --cov-config={toxinidir}/setup.cfg {posargs}
     double: python -c 'import sys; from astropy import test; test(); sys.exit(test())'
 
-[testenv:build_docs]
-changedir = docs
-description = invoke sphinx-build to build the HTML docs
-extras = docs
-commands =
-    pip freeze
-    sphinx-build -W -b html . _build/html
```

Comment:
We should keep this as well.

I am +1 on removing the `-W` here to fix issues with building docs on mac, but I suggest we actually punt that discussion to another PR / issue.

Reply:
Ah, okay. So we keep this so that developers can use tox to build docs but *not* use it in CI, correct?

Reply:
Anyways, I reverted the deletion as you requested.